api/types/apitypes.go

```diff
@@ -185,28 +185,29 @@ type TxInputID struct {
 	Index uint32 `json:"vin_index"`
 }
 
-// AtomicSwapTx models an atomic swap contract OR redemption tx, including
-// information about all completed swaps related to the transaction.
-type AtomicSwapTx struct {
-	TxID  string        `json:"tx_id"`
-	Type  string        `json:"type"`
-	Swaps []*AtomicSwap `json:"swaps"`
+// TxAtomicSwaps defines information about completed atomic swaps that are
+// related to a transaction.
+type TxAtomicSwaps struct {
+	TxID        string                 `json:"tx_id"`
+	Found       string                 `json:"found"`
+	Contracts   map[uint32]*AtomicSwap `json:"contracts,omitempty"`
```

Comment:
Named this `Contracts` instead of `Swaps` because this property contains references to txouts that define atomic swap contracts, just as `Redemptions` and `Refunds` link txins to contract redeems/refunds.